src/main/java/de/teamlapen/vampirism/entity/vampire/BasicVampireEntity.java

```diff
@@ -348,7 +346,7 @@ protected void registerGoals() {
         this.goalSelector.addGoal(1, new BreakDoorGoal(this, (difficulty) -> {
             return difficulty == net.minecraft.world.Difficulty.HARD;
         }));//Only break doors on hard difficulty
-        this.tasks_avoidHunter = new AvoidEntityGoal(this, CreatureEntity.class, 10, 1.0, 1.1, VampirismAPI.factionRegistry().getPredicate(getFaction(), false, true, false, false, VReference.HUNTER_FACTION));
+        this.tasks_avoidHunter = new AvoidEntityGoal<CreatureEntity>(this, CreatureEntity.class, 10, 1.0, 1.1, VampirismAPI.factionRegistry().getPredicate(getFaction(), false, true, false, false, VReference.HUNTER_FACTION));
```

Comment:
Usually it is best to just use empty brackets <> for generics. Saves some typing when changing things and removes redundancy. Doesn't hurt though